java/src/jmri/jmrit/display/layoutEditor/LayoutEditor.java

```diff
@@ -857,15 +857,15 @@ public void popupMenuCanceled(PopupMenuEvent e) {
         rotationPanel.add(rotationLabel);
         rotationPanel.add(rotationComboBox);
 
-        zoomPanel.add(new JLabel(Bundle.getMessage("ZoomLabel") + ":"));
+        zoomPanel.add(new JLabel(String.format("{}:", Bundle.getMessage("ZoomLabel"))));
```

Comment:
To support I18N there's a Bundle.getMessage("MakeLabel", "your label without colon") at the Njmri.NBB base level that e.g. adds a space in front of the colon in French.

Reply:
Awesome! I'll use that… next commit.